File: app/src/main/java/com/moggot/findmycarlocation/data/model/route/Leg.java
package com.moggot.findmycarlocation.data.model.route;

import com.google.gson.annotations.SerializedName;

public class Leg {

    @SerializedName("distance")
    private Distance distance;
    @SerializedName("duration")
    private Duration duration;

    @SerializedName("steps")

    public Distance getDistance() {
        return distance;
    }

    public Duration getDuration() {
        return duration;
    }
}

File: app/src/main/java/com/moggot/findmycarlocation/data/model/route/Leg.kt
package com.moggot.findmycarlocation.data.model.route

import com.squareup.moshi.Json

data class Leg(
    @field:Json(name = "distance") val distance: Distance? = null,
    @field:Json(name = "duration") val duration: Duration? = null,
    @field:Json(name = "start_address") val startAddress: String? = null,
    @field:Json(name = "end_address") val endAddress: String? = null
)
